pages/_app.tsx

```diff
@@ -60,35 +75,49 @@ function DiscordDocsApp({ Component, pageProps, router }: AppProps) {
 
   return (
     <AuthenticationContextProvider>
-      <NextNprogress color="#EB459E" />
-      <Wrapper>
-        <Head>
-          <title>{title}</title>
-          <meta name="description" content={description} />
-          <meta name="og:title" content={title} />
-          <meta name="og:description" content={description} />
-        </Head>
-        {dontRenderSidebarOn.includes(router.pathname) ? undefined : (
-          <Sidebar
-            items={[
-              ...DEFAULT_SIDEBAR_ITEMS,
-              ...(pageProps.sidebarItems || []),
-            ]}
-          />
-        )}
-
-        {hideLoginButtonOn.includes(router.pathname) ? (
-          <></>
-        ) : (
-          <AccountHeaderContainer>
-            <AccountHeader />
-          </AccountHeaderContainer>
-        )}
-
-        <ContentWrapper>
-          <Component {...pageProps} />
-        </ContentWrapper>
-      </Wrapper>
+      <ThemeProvider
+        disableTransitionOnChange
+        attribute="class"
+        value={{ light: lightTheme.className, dark: "dark-theme" }}
+        defaultTheme="system"
+      >
+        <NextNprogress color="#5865f2" />
```

Comment:
Are we sure we want to use the primary brand colour for the loader? It is virtually invisible on the summaries page when the user navigates to one of the summaries.

Reply:
It should be a color within our theme

Reply:
So should we add a new colour to the theme? The fuchsia colour was selected directly from Discord's branding guidelines (and thus fits with the current blurple palette)

Reply:
Fair point, I mean im fine with it, going to cc @goverfl0w and @Liege72 for their opinion